client/app/bundles/course/assessment/submission/reducers/submissionFlags.js

```diff
@@ -29,6 +29,7 @@ export default function (state = initialState, action) {
     case actions.PUBLISH_REQUEST:
     case actions.DELETE_FILE_REQUEST:
     case actions.IMPORT_FILES_REQUEST:
+    case actions.GET_PAST_ANSWERS_REQUEST:
```

Comment:
Since i'm sending a request to the server, and it might take a moment to load, its used to disable the other buttons on the screen so that they don't send another request before the past answers have loaded.